components/shooter.py

```diff
@@ -57,6 +72,17 @@ def execute(self) -> None:
             self.loading_piston.startPulse()
             self.inject = False
 
+        self.led.setData(self.led_vals)
+
+    def set_led_speed_RGB(self, r, g, b) -> None:
```

Comment:
I think this would be more descriptive if this was called `get_flywheel_led` or `get_shooter_led`.